plugins/00_dokku-standard/commands

```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env bash
 set -eo pipefail; [[ $DOKKU_TRACE ]] && set -x
-source "$PLUGIN_PATH/common/functions"
-source "$PLUGIN_PATH/config/functions"
+source "$PLUGIN_CORE_AVAILABLE_PATH/common/functions"
+source "$PLUGIN_CORE_AVAILABLE_PATH/config/functions"
```

Comment:
So should we be sourcing from `$PLUGIN_CORE_AVAILABLE_PATH` when sourcing anything not in common?